### PR TITLE
Upgrade version of bundler

### PR DESCRIPTION
**Pull request by alexlrobertson**
https://app.clubhouse.io/bonusly/story/9647/upgrade-bundler-to-version-2-1-0-or-later-in-filereaderjs

### CODE REVIEW

**Review by ehinson — APPROVED** [review]
Looks good to me

File: filereaderjs.gemspec [hunk 1]
```diff
@@ -18,6 +18,6 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.11"
+  spec.add_development_dependency 'bundler', '~> 2.2.18'
```
Comment (ajaswa) [inline filereaderjs.gemspec]: Why the change in quotes?

**Review by ajaswa — APPROVED** [review]

File: filereaderjs.gemspec (hunk 1 above)
Comment (alexlrobertson) [reply filereaderjs.gemspec]: This was copied from https://github.com/bonusly/tangolicious/pull/1
